Makefile

```diff
@@ -245,5 +245,12 @@ $(eval $(call build-package,php,8.1.13-r0))
 $(eval $(call build-package,composer,2.4.4-r0))
 $(eval $(call build-package,docker-credential-ecr-login,0.6.0-r1))
 $(eval $(call build-package,wait-for-it,0.20200823-r1))
+$(eval $(call build-package,libunistring,1.1-r0))
+$(eval $(call build-package,gettext,0.21.1-r0))
+$(eval $(call build-package,utmps,0.1.2.0-r0))
```

Comment:
utmps is not needed on wolfi, as glibc supports utmp.

Reply:
ok great - lemme remove utmps